src/components/PluginCapabilitiesSection.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div>
-    <h2 id="capabilities">Capabilities</h2>
+    <p class="text-3xl py-4" id="capabilities">Capabilities</p>
```

Comment:
Is it standard to not use any header labels or headset classes at all?

Reply:
with preflight headings are un-styled by default: https://tailwindcss.com/docs/preflight#headings-are-unstyled

tbh - I've only ever worked in configs where someone else had already added them as part of a more complex custom base style, and for my random side project's i've just always used the tailwind typography plugin. 

tl;dr this is fine I think until we have time to extend the base style?

Reply:
I don't have a strong opinion here but if we wanted to keep this DRY and ensure consistency but don't think this merits an actual component, I think tailwind supports a middle path option within its plugin system where we could factor `text 3xl py-4` into a class: https://tailwindcss.com/docs/functions-and-directives#using-apply-with-per-component-css.

That said, I think the approach @rwfeather took here is the approach that tailwind recommends.